vertical-pod-autoscaler/recommender/recommender.go

```diff
@@ -146,6 +146,44 @@ func (r *recommender) loadPods() {
 	}
 }
 
+func (r *recommender) loadMetrics() {
```

Comment:
I'd suggest "loadRealTimeMetrics()", to better explain what's going on, especially that this is different from loading history (of the metrics).

Reply:
ready.